### PR TITLE
Certificate processing service initial implementation

### PR DESCRIPTION
**Pull request by NitaiBanik**

### CODE REVIEW

File: src/Services/CertificateRegistration/CertificateRegistration.API/Startup.cs [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        }
+
+        // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
+        public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+        {
+            if (env.IsDevelopment())
+            {
+                app.UseDeveloperExceptionPage();
+                app.UseSwagger();
+                app.UseSwaggerUI(c => c.SwaggerEndpoint("/swagger/v1/swagger.json", "CertificateRegistration.API v1"));
```
Comment (ratanparai) [inline src/Services/CertificateRegistration/CertificateRegistration.API/Startup.cs:44]: Move this outside of the if block

File: src/Services/CertificateRegistration/CertificateRegistration.API/Startup.cs [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            }
+
+            app.UseHttpsRedirection();
```
Comment (ratanparai) [inline src/Services/CertificateRegistration/CertificateRegistration.API/Startup.cs:47]: we do not have https yet. so please remove it.

**Review by ratanparai — CHANGES_REQUESTED** [review]

File: src/Services/CertificateRegistration/CertificateRegistration.API/Startup.cs [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+            });
+        }
+
+        // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
+        public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+        {
+            if (env.IsDevelopment())
+            {
+                app.UseDeveloperExceptionPage();
+                app.UseSwagger();
```
Comment (ratanparai) [inline src/Services/CertificateRegistration/CertificateRegistration.API/Startup.cs:43]: You also need to move the `app.UseSwagger()` outside of development block

**Review by ratanparai — CHANGES_REQUESTED** [review]

**Comment by ratanparai** [comment]
@NitaiBanik: can you please rename the service to `CertificateProcessing `service?

**Comment by NitaiBanik** [comment]
> @NitaiBanik: can you please rename the service to `CertificateProcessing `service?

File: src/Services/CertificateProcessing/CertificateProcessing.API/Startup.cs [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+        // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
+        public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+        {
+            if (env.IsDevelopment())
+            {
+                app.UseDeveloperExceptionPage();
+            }
+
+             app.UseSwaggerUI(
```
Comment (ratanparai) [inline src/Services/CertificateProcessing/CertificateProcessing.API/Startup.cs:38]: Please add `app.UseSwagger();`

Also, please fix the spacing.

File: src/Services/CertificateProcessing/CertificateProcessing.API/Controllers/HelloWorldController.cs [hunk 5]
```diff
@@ -0,0 +1,15 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace CertificateProcessing.API.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class HelloWorldController : ControllerBase
+    {
+        [HttpGet]
+        public static string Ping()
```
Comment (ratanparai) [inline src/Services/CertificateProcessing/CertificateProcessing.API/Controllers/HelloWorldController.cs:10]: Static will not work here. Better would be -

```cs
public ActionResult<string> Ping()
{
  return Ok("Pong");
}
```

**Review by ratanparai — CHANGES_REQUESTED** [review]
Thank you for the PR. I will create dockerfile for the service after you fix those

File: src/Services/CertificateProcessing/CertificateProcessing.API/Startup.cs (hunk 4 above)
Comment (NitaiBanik) [reply src/Services/CertificateProcessing/CertificateProcessing.API/Startup.cs]: I Hope, the PR is ok now.

**Comment by ratanparai** [comment]
@NitaiBanik: I tried to push Dockerfile modifications, but I do not have access to your branch. Can you please give me permission? Please check https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork for more info

**Review by ratanparai — APPROVED** [review]

**Pull request merged**